src/benchmark_register.cc

```diff
@@ -124,6 +156,10 @@ bool BenchmarkFamilies::FindBenchmarks(
     // Family was deleted or benchmark doesn't match
     if (!family) continue;
 
+    if (auto p_fixture_benchmark = dynamic_cast<FixtureBenchmark*>(family.get())) {
```

Comment:
if you need a p_ to indicate it's a pointer, maybe use `auto*` instead. ie, prefer types to naming to indicate, well, type.

Reply:
I will change the code.